Library/Formula/dshb.rb

```diff
@@ -0,0 +1,19 @@
+class Dshb < Formula
+  desc "OS X system monitor in Swift"
+  homepage "https://github.com/beltex/dshb"
+  url "https://github.com/beltex/dshb/releases/download/v0.0.5/dshb-0.0.5-source.zip"
+  sha256 "c1b65114feedc67030fbc93bc8a76482dff6c46eb8055e7ccbbc92acb1c75e01"
+
+  depends_on :xcode => ["6.3", :build]
+
+  def install
+    system "make", "release"
+    bin.install "bin/dshb"
+    man1.install "doc/dshb.1"
+  end
+
+  test do
+    pipe_output("#{bin}/dshb", "q")
+    assert $?.success?
```

Comment:
You can pass a third argument to pipe_output which is the exit status to assert, like `pipe_output("#{bin}/dshb", "q", 0)`